assets/css/_route_ladder.scss

```diff
@@ -1,6 +1,6 @@
-.m-route-ladder {
-  display: flex;
-  flex-direction: column;
+.m-route-ladder__header-and-controls {
```

Comment:
What if the header and the controls were separate rows of the grid, instead of having to be wrapped together?